File: In_C_Programming/q10.c
#include <stdio.h>

int copy_Elements(int arr[], int temp[], int length)
{

    for (int i = 0; i < length; i++)
    {
        temp[i] = arr[i];
    }

    printf("The Copied Version is: \n");

    for (int i = 0; i < length; i++)
    {
        printf("%d \n", temp[i]);
    }

    return 0;
}

int main()
{

    int size;

    printf("Enter The Array Length: \n");
    scanf("%d", &size);

    int x[size];
    int y[size];

    printf("Enter The Elements: \n");

    for (int i = 0; i < size; i++)
    {
        scanf("%d", &x[i]);
    }

    copy_Elements(x, y, size);

    return 0;
}

File: In_Java_Programming/q10.java
import java.util.Scanner;

public class q10 {

    public static void callCopy(int arr[], int temp[], int ln) {

        for (int i = 0; i < ln; i++) {

            temp[i] = arr[i];
        }

        System.out.println("The Copied Array is: ");

        for (int i = 0; i < ln; i++) {
            System.out.print(temp[i] + " ");
        }
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        int arr[] = new int[ln];
        int temp[] = new int[ln];

        for (int i = 0; i < args.length; i++) {

            arr[i] = sc.nextInt();
        }

        callCopy(arr, temp, ln);

        sc.close();
    }
}
